ui/client/actions/notificationActions.js

```diff
@@ -1,13 +1,14 @@
 import React from "react"
 import Notifications from 'react-notification-system-redux'
 import InlinedSvgs from "../assets/icons/InlinedSvgs"
-import Notification from "../components/notifications/Notification";
+import Notification from "../components/notifications/Notification"
+import uuid from "react-tabs/lib/helpers/uuid"
```

Comment:
In other places we use - import {v4 as uuid4} from "uuid";

Reply:
Done